src/sidebar/components/test/group-list-item-out-of-scope-test.js

```diff
@@ -1,106 +1,98 @@
 'use strict';
 
-const angular = require('angular');
+const { mount } = require('enzyme');
+const preact = require('preact');
+const { createElement } = require('preact');
 const proxyquire = require('proxyquire');
-const util = require('../../directive/test/util');
 
 const { events } = require('../../services/analytics');
 
-describe('groupListItemOutOfScope', () => {
+describe('GroupListItemOutOfScope', () => {
   let fakeAnalytics;
   let fakeGroupListItemCommon;
+  let GroupListItemOutOfScope;
+
+  const fakeGroup = {
+    id: 'groupid',
+    links: {
+      html: 'https://hypothes.is/groups/groupid',
+    },
+    logo: 'dummy://hypothes.is/logo.svg',
+    organization: { name: 'org' },
+  };
+
+  // Click on the item to expand or collapse it.
+  const toggle = wrapper =>
+    wrapper
+      .find('div')
+      .first()
+      .simulate('click');
 
   before(() => {
     fakeGroupListItemCommon = {
       orgName: sinon.stub(),
       trackViewGroupActivity: sinon.stub(),
     };
 
-    // Return groupListItemOutOfScope with groupListItemCommon stubbed out.
-    const groupListItemOutOfScope = proxyquire(
-      '../group-list-item-out-of-scope',
-      {
-        '../util/group-list-item-common': fakeGroupListItemCommon,
-        '@noCallThru': true,
-      }
-    );
+    GroupListItemOutOfScope = proxyquire('../group-list-item-out-of-scope', {
+      // Use same instance of Preact module in tests and mocked module.
+      // See https://robertknight.me.uk/posts/browserify-dependency-mocking/
+      preact,
 
-    angular
-      .module('app', [])
-      .component('groupListItemOutOfScope', groupListItemOutOfScope);
+      '../util/group-list-item-common': fakeGroupListItemCommon,
+      '@noCallThru': true,
+    });
   });
 
   beforeEach(() => {
     fakeAnalytics = {
       track: sinon.stub(),
       events,
     };
-
-    angular.mock.module('app', { analytics: fakeAnalytics });
   });
 
   const createGroupListItemOutOfScope = fakeGroup => {
-    return util.createDirective(document, 'groupListItemOutOfScope', {
-      group: fakeGroup,
-    });
+    return mount(
+      <GroupListItemOutOfScope analytics={fakeAnalytics} group={fakeGroup} />
+    );
   };
 
-  it('calls groupListItemCommon.trackViewGroupActivity when trackViewGroupActivity is called', () => {
-    const fakeGroup = { id: 'groupid' };
+  it('calls trackViewGroupActivity when "Go to group page" link is clicked', () => {
```

Comment:
This seems like duplicate functionality with the `group-list-item` component, yes? Perhaps on the list for a down-the-road refactor?

Reply:
Yes, this will most likely change quite soon, pending the ongoing design exploration around this menu in the #design channel.